### ISSUE TITLE
Add rule for detecting unreachable code

### ISSUE THREAD

**Issue by mnojek** [MarketSquare/robotframework-robocop#810]
If `RETURN`, `BREAK` or `CONTINUE` are used, the code after them (in the same scope) is not reachable. We should detect such situations and report them as `unreachable-code`.

For example:
```
Example
    FOR    ${animal}    IN    cat    dog
        IF    '${animal}' == 'cat'
            CONTINUE
            Log  ${animal}  # unreachable code
        END
    END
    RETURN
    Log    Unreachable log
```

**Comment by bhirsz** [comment]
It partly overlaps with keyword-after-return rule. Only partly, because it only checks if there is keyword after Return From Keyword and looking on implementation it appears to be buggy. But it could be a good place to store this rule.

**Comment by bhirsz** [comment]
My bad - I though it didn't handle following code:

```
*** Keywords ***
Keyword
    IF    $condition
	    Return From Keyword
	END
	Step

```

but it works fine.
